catalog/ui/catalog-ui-search/src/main/webapp/component/workspace-lists/workspace-lists.view.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define, setTimeout*/
+var Marionette = require('marionette');
+var CustomElements = require('js/CustomElements');
+var template = require('./workspace-lists.hbs');
+var store = require('js/store');
+var ListSelectorView = require('component/dropdown/list-select/dropdown.list-select.view');
+var DropdownModel = require('component/dropdown/dropdown');
+var ResultSelectorView = require('component/result-selector/result-selector.view');
+var $ = require('jquery');
+var ListCreateView = require('component/list-create/list-create.view');
+var PopoutView = require('component/dropdown/popout/dropdown.popout.view');
+
+let selectedListId;
```

Comment:
But if we never transition then we'll be stuck in the past forever.

Reply:
With that same logic we should not be using Arrow functions either since most of the existing codebase uses old style logic.

Reply:
We should look at transitioning to `let` and `const` though.  No reason to not do so in new files like this one.

https://stackoverflow.com/questions/40058571/is-there-a-reason-not-to-replace-var-with-let-in-javascript

There are some sticky points with doing a massive find / replace unfortunately, like defining a `var` inside of an `if` statement.  I don't think we are doing that a whole lot, if at all.  It would be worth trying to do a massive find / replace and seeing if things just work.

Reply:
I would very much like to use more ES6; I just suggested changing that `let` since the decision seemed to be to use `var` generally for consistency and I wasn't sure where we stood on ES6 compatibility. Using `let` and `const` everywhere would be even better, though. _Down with `var`!_

If we have full ES6 capabilities, then at the very least, all the new code in this pull request should use `let` or `const` rather than `var`. Dealing with preexisting ES5 code with `var` is something we should discuss... preferably sooner rather than later.